### DISCUSSION TITLE
Route-Validation in Nuxt3

### DISCUSSION THREAD

**Discussion by kaikaibenkai** [nuxt/nuxt#19063] [Questions] [answered]
[On Stackoverflow](https://stackoverflow.com/questions/75449446/route-validation-in-nuxt3)

---

Update: **The bug has been sloved in v3.2.1**

**Comment by TheAlexLichter** [comment, marked as answer]
See https://github.com/nuxt/nuxt/issues/18567

**Reply by kaikaibenkai** [reply]
Thanks for your anwser!
